### PR TITLE
Include new stations_by_status page

### PR DESCRIPTION
**Pull request by davidfokkema**
Include a new station page, grouped by status, as per @153957's suggestion.

### CODE REVIEW

**Comment by davidfokkema** [comment]
Looks like this:

screen shot 2018-07-05 at 12 58 03

File: publicdb/status_display/templates/stations_by_status.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+{% block current_status %}currentPage{% endblock %}
+
+{% block header %}
+    List of HiSPARC stations, by status
+{% endblock %}
+
+{% block list %}
+    {% for status_group, stations in station_groups.items %}
+      <h2>{{ status_group | capfirst }}</h2>
```
[the remark this answers is not in the document]
Comment (davidfokkema) [reply publicdb/status_display/templates/stations_by_status.html]: Nice! Fixed in 40a0e88.

**Comment by davidfokkema** [comment]
screen shot 2018-07-05 at 13 35 08

**Comment by 153957** [comment]
Cool!

**Pull request merged**